build-system/global-configs/client-side-experiments-config.json

```diff
@@ -23,6 +23,18 @@
     {
       "name": "story-load-inactive-outside-viewport",
       "percentage": 1
+    },
+    {
+      "name": "amp-sticky-ad-to-amp-ad-v2",
+      "percentage": 0.05
+    },
+    {
+      "name": "story-ad-page-outlink",
+      "percentage": 1,
```

Comment:
Is your intention to launch this, or to experiment?

Reply:
Experiment first with this PR, (then eventually launch).